cmd/faktory-cli/repl.go

```diff
@@ -77,24 +78,50 @@ func main() {
 func repl(path string, store storage.Store) {
 	fmt.Printf("Using RocksDB %s at %s\n", gorocksdb.RocksDBVersion(), path)
 
-	rdr := bufio.NewReader(os.Stdin)
+	var completer = readline.NewPrefixCompleter(
+		readline.PcItem("version"),
+		readline.PcItem("flush"),
+		readline.PcItem("backup"),
+		readline.PcItem("restore"),
+		readline.PcItem("repair"),
+		readline.PcItem("purge"),
+		readline.PcItem("exit"),
+		readline.PcItem("help"),
+	)
+
+	l, err := readline.NewEx(&readline.Config{
+		Prompt:          "> ",
+		HistoryFile:     "/tmp/faktory.tmp",
```

Comment:
I would change this to `$HOME/.faktory/cli.history`

Reply:
Yeah, this makes much more sense. All files will be together this way.